src/functionalTest/resources/scenarios/RIA-2573-ftpa-appeal-decision-allowed-resident-judge.json

```diff
@@ -0,0 +1,66 @@
+{
+  "description": "RIA-2573 FTPA Applicant Application Decision Allowed - Resident Judge",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "Judge",
+    "input": {
+      "id": 1002,
+      "eventId": "residentJudgeFtpaDecision",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ftpaAppellantDecisionOutcomeType": "allowed",
+          "ftpaAppellantSubmitted": "Yes",
+          "ftpaRespondentSubmitted": "No",
+          "notificationsSent": []
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "notificationsSent": [
+          {
+            "id": "1002_FTPA_APPLICATION_DECISION_HOME_OFFICE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "1002_FTPA_APPLICATION_DECISION_LEGAL_REPRESENTATIVE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ]
+      }
+    },
+    "notifications": [
+      {
+        "reference": "1002_FTPA_APPLICATION_DECISION_HOME_OFFICE",
+        "recipient": "{$respondentEmailAddresses.respondentEvidenceDirection}",
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal",
+        "body": [
+          "HMCTS reference: PA/12345/2019",
```

Comment:
You need to remove static content "HMCTS reference:" (and similar) from these functional tests.

Reply:
Fixed